### DISCUSSION TITLE
how to deal with sales kit

### DISCUSSION THREAD

**Discussion by mabroens** [inventree/InvenTree#6458] [Q&A]
Hi team,
Suppose you have 3 product (each of which we require build/purchase orders) for example 2 PCBA's (BO), 1 wiring harnass (PO), one housing(BO), 1x testing and 1x Licence (Non trackable) and i would like to add this as a single sales kit.
but when selling the kit it should obviously take the parts from stock.

is this possible and if so how do i set this up. or even better is there an example in the test db's that shows this.

Currenty i created a kit and added the diffrent parts to its bom. is this the way to go?
and finally i would like to print each BOM item on the sales order as sub parts from the sales kit.

Br Martijn

**Comment by miggland** [comment]
Relevant to your question:

https://github.com/inventree/InvenTree/issues/2624

I currently do what you mention: the set part has a BoM of the parts it includes. When I need it, I create a build order so that the stock is transferred, then sell that. It's a very manual process, but the only alternative I know of (and I don't need it very often)

**Comment by SergeoLacruz** [comment]
We do it tha same. This is a product and has a BOM incluging box, label, plastic bag, cable, user guide, ...
This is needed bacause we also need stock tracking of those parts. 
A BO might contain 20 pices and parts can be finished after packing.

**Comment by mabroens** [comment]
is it also possible to print the BOM list on the invoice? or is that just like printing a BOM for a build order? did not try yet.

but i would like to have the individual parts printed on a shipping list or pick list.

**Comment by SergeoLacruz** [comment]
As long as all required info is somewhere in Inventree you can generate whatever you like using the report system. It is very powerfull. You can create an invoice template for each SO. In the SO there is the part and in the part the BOM. If you have all addresses of your customers in  Inventree you can also generate a complete invoice.
